superset/models/slice.py

```diff
@@ -234,10 +234,7 @@ def data(self) -> Dict[str, Any]:
 
     @property
     def digest(self) -> str:
-        """
-        Returns a MD5 HEX digest that makes this dashboard unique
-        """
-        return md5_sha_from_str(self.params or "")
+        return get_chart_digest(self)
```

Comment:
don't see any other uses of `get_chart_digest` why not just post it's contents here?

Reply:
Good idea 👍